AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginIntegrationTests/SubscriptionEndToEndTests.swift

```diff
@@ -103,7 +103,7 @@ class SubscriptionEndToEndTests: SyncEngineIntegrationTestBase {
             XCTFail("Listener not registered for hub")
             return
         }
-        sendUpdateRequest(forId: id, content: updatedContent, version: 1)
+        try await sendUpdateRequest(forId: id, content: updatedContent, version: 1)
```

Comment:
This test seems to run indefinitely on my machine. Could you please confirm?

Reply:
refactored this test to use `asyncExpectation`, let's see if this helps, it seems to run fine on Xcode 14.6